### PR TITLE
#485 update new cr fields

### PR DESCRIPTION
**Pull request by syedDS**
new C_R update from initiation to activation on type

### CODE REVIEW

**Comment by syedDS** [comment]
The code tested on the local repo didnt gave any error but is giving after PR , can someone review it

**Comment by jamescd18** [comment]
It looks like you changed the `package.json` and `package-lock.json` files, both of which you do not want to be changing. This potentially is part of why it broke.

File: src/components/change-requests/new-change-request/new-change-request-page/common-form-fields/common-form-fields.tsx [hunk 1]
```diff
@@ -35,7 +36,9 @@ const CommonFormFields: React.FC = () => {
       <div className={'px-4'}>
         Type
         <Form.Control as="select" custom>
-          {['New Function', 'Design Issue', 'Other', 'Initiation', 'Stage Gate'].map((t) => (
+          {Object.values(ChangeRequestType).map((
+            t // change from intiation -> Activation
```
Comment (jamescd18) [inline src/components/change-requests/new-change-request/new-change-request-page/common-form-fields/common-form-fields.tsx:40]: This comment looks like it's specific to this pull request so you'll want to remove it

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Awesome! Just one small change for this code. The bigger thing is that this PR has a very messy commit history and I don't see any of the commits being tagged with the ticket number. Given how small the code change is, I would love for you to create a fresh new branch, make just 1 commit that has the ticket number as described in the contributor guide, and open a new PR. I think this is a really good learning experience because this PR here allowed you to figure out how things work and get feedback, and then the fresh new PR will be the one that we actually approve and merge into main. If you have any questions or if anything is confusing, reach out to your tech lead on Slack. Thank you!

**Pull request closed without merging**